manifests/04-deployment.yaml

```diff
@@ -20,7 +20,6 @@ spec:
       labels:
         app: machine-approver
     spec:
-      hostNetwork: true
```

Comment:
this is probably fine, at least from our point of view.

Reply:
otherwise, this will depend on SDN networking which might be undesired.

Reply:
This is the only place where CSR approval flow will experience any changes. We don't use `KUBERNETES_SERVICE_HOST` explicitly anywhere, so as `127.0.0.1`. I wonder if there is a way to check the health of the API server before starting `CSR` serving.

Reply:
LBs watch `/readyz` endpoint as described in https://github.com/openshift/installer/blob/master/docs/dev/kube-apiserver-health-check.md so maybe this is something the operator could rely on as well.

Wondering what would this operator do when the local API server rolls out to a new version?

Reply:
why 127.0.0.1 in the first place? This feels wrong. Don't use it.

Reply:
@p0lyn0mial As it's late in the release cycle can we keep hostNetwork to reduce the scope of the change and  move forward with the PR?

Reply:
@p0lyn0mial friendly ping :) so we can close anything open for 4.6

Reply:
yes, sorry, updating the PR.

Reply:
done :)